http/polytracker/main.py

```diff
@@ -0,0 +1,119 @@
+import os
+from pathlib import Path
+import signal
+import sys
+import subprocess
+from tempfile import NamedTemporaryFile
+from typing import List, Optional
+
+from polytracker.clustering import load_indexes_for_matching, MultipleTaintSourcesError, ordered_edit_distance
+from polytracker.clustering import match as cluster_match
+
+DEFAULT_APACHE_PORT = 56781
+
+def run_httpd(request_path: str, output_path: str, apache_port: int = DEFAULT_APACHE_PORT, timeout: Optional[float] = None):
+    if not (apache_port > 0 and apache_port < 65536):
+        raise ValueError("Invalid port number")
+    try:
+        subprocess.check_call(["/usr/bin/httpd_harness", request_path], env={
+            "POLYDB": output_path,
+            "POLYTRACKER_STDOUT_SINK": "1",
+            "APACHE_PORT": apache_port
```

Comment:
`apache_port` is an `int`, so this should be `"APACHE_PORT": str(apache_port)`. As written, the script crashes with a TypeError.

Reply:
```suggestion
            "APACHE_PORT": str(apache_port)
```